### ISSUE TITLE
Remove unused DISPATCH_IMAGE build arg from docker-compose.yml

### ISSUE THREAD

**Issue by TheDahv** [Netflix/dispatch-docker#5]
The docker-compose file describes a configurable argument for `DISPATCH_IMAGE`: https://github.com/Netflix/dispatch-docker/blob/master/docker-compose.yml#L8-L9

It appears this isn't used by the associated image at all.

```
dispatch-docker <master> $ DISPATCH_IMAGE=dispatch-local docker-compose build web 

...

[Warning] One or more build-args [DISPATCH_IMAGE] were not consumed
Successfully built fa22fc260b63
Successfully tagged dispatch-local:latest
```

Indeed, the [Dockerfile](https://github.com/Netflix/dispatch/blob/develop/docker/Dockerfile) this configuration points to makes no mention of that term.

This appears to be unused configuration and a red herring to a developer trying to troubleshoot issues. It should probably be removed.

**Comment by kevgliss** [comment]
Agreed, will get it removed.

**Comment by karl-johan-grahn** [comment]
Same thing with `DISPATCH_DOMAIN`, doesn't seem to be used at all

**Comment by kevgliss** [comment]
Fixed in #23, thanks again.
